src/main/java/org/mamute/model/User.java

```diff
@@ -357,12 +354,12 @@ public boolean isAuthorOf(Votable votable) {
 		return id == votable.getAuthor().getId();
 	}
 	
-	public boolean hasKarmaToAnswerOwnQuestion() {
-		return (this.karma >= PermissionRulesConstants.ANSWER_OWN_QUESTION) || isModerator(); 
+	public boolean hasKarmaToAnswerOwnQuestion(long answerOwnQuestion) {
+		return (this.karma >= answerOwnQuestion) || isModerator();
```

Comment:
I prefer the other approach, passing the `EnvironmentKarma` as argument. The way it is, someone can simply pass 0 to this method and it is the same for the method below